x-pack/plugins/security_solution/cypress/tasks/alerts_detection_rules.ts

```diff
@@ -186,7 +186,7 @@ export const filterByDisabledRules = () => {
 };
 
 export const goToRuleDetails = () => {
-  cy.get(RULE_NAME).first().click();
+  cy.get(RULE_NAME).first().click({ force: true });
```

Comment:
Nit: do we have to have this `force: true` here? I think we're moving towards trying to get rid of use of this in the future.

Reply:
ah we probably do not need the force true. Thanks!

Reply:
removed here: [`2515470` (#162839)](https://github.com/elastic/kibana/pull/162839/commits/2515470b76907503c91d450e64630872ab6dbf1f)